truffle.js

```diff
@@ -1,38 +1,41 @@
 module.exports = {
-  networks: {
-    development: {
-      host: "localhost",
-      network_id: "*", // Match any network id
-      port: 8545,
-      gas: 12000000,
-      gasPrice: 0x01,
-    },
-    integration: {
-      host: "localhost",
-      network_id: "*",
-      port: 8546,
-      gas: 9000000,
-      gasPrice: 0x3B9ACA00,
-    },
-    coverage: {
-      host: "localhost",
-      network_id: "*",
-      port: 8555,         // <-- If you change this, also set the port option in .solcover.js.
-      gas: 0xfffffffffff, // <-- Use this high gas value 
-      gasPrice: 0x01,     // <-- Use this low gas price
+    networks: {
+        development: {
+            host: 'localhost',
+            network_id: '*', // Match any network id
+            port: 8545,
+            gas: 12000000,
```

Comment:
This should be changed to 8 million. For an example, if contract deployment cost exceeds 8 million, the unit test should fail 💭  
8 million is block gas limit on main-net and rospten.

Reply:
I don't think that is in the scope of this ticket 🤔 I am afraid to break stuff.

Reply:
Ideally, it should not break, but If it breaks we will get a chance to fix it, or create a ticket. Atleast we will know.

Reply:
OK I updated it. Let's see if CI passes 😱